src/Nest/XPack/MachineLearning/DeleteExpiredData/ElasticClient-DeleteExpiredData.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	public partial interface IElasticClient
+	{
+		/// <summary>
+		/// Deletes expired data for Machine Learning.
+		/// </summary>
+		IDeleteExpiredDataResponse DeleteExpiredData(Func<DeleteExpiredDataDescriptor, IDeleteExpiredDataRequest> selector = null);
```

Comment:
Yes, I think it's fine to include this.

It gets called automatically once per day per cluster, and we didn't document it because that should be enough. But if people really want to call it more often it doesn't hurt.